src/app/modules/submission/email-submission/email-submission.routes.ts

```diff
@@ -29,6 +29,6 @@ export const EmailSubmissionRouter = Router()
 EmailSubmissionRouter.post(
   '/:formId([a-fA-F0-9]{24})',
   limitRate({ max: rateLimitConfig.submissions }),
-  CaptchaFactory.validateCaptchaParams,
+  CaptchaMiddleware.validateCaptchaParams,
   handleEmailSubmission,
```

Comment:
This can eventually be merged right? Or are we keeping it this way so it will be easy to rip out if necessary?

Reply:
ah hmm. good point, let me merge it now.

Reply:
addressed in https://github.com/opengovsg/FormSG/pull/2157/commits/e41e80fd5ca7a84adf75da1dab2499df5883786b